src/main/app/authorized-axios/AuthorizedAxios.ts

```diff
@@ -104,9 +104,9 @@ export class AuthorizedAxios extends Axios {
 
   private requestInterceptor = () => {
     this.interceptors.request.use(config => {
-      if (this.oauth.token && config.url !== '/o/token') {
```

Comment:
Do we want to make sure this.oauth.token is not null, just in case?

Reply:
Addressed in https://github.com/hmcts/idam-user-dashboard/pull/449/commits/86496c9403d3f567651a8b82689bbc34f1afd550